CONTRIBUTING.md

```diff
@@ -95,6 +95,16 @@ Portainer can now be accessed at <https://localhost:9443>.
 
 Find more detailed steps at <https://documentation.portainer.io/contributing/instructions/>.
 
+### Build customisation
+
+By default, `yarn start` will use `/tmp/portainer/` for the data store, so it won't persist over reboots.
```

Comment:
this comment seems to be specific to the first item in the settings list only, isn't it? might be clearer to move it over there

Reply:
agreed - this was written ages ago, and rolled into this PR :)

off to update them